### ISSUE TITLE
Latest version of Pretty Diff published to NPM

### ISSUE THREAD

**Issue by prettydiff** [Unibeautify/beautifier-prettydiff#114]
Published to the old namespace - https://www.npmjs.com/package/prettydiff

The option names have changed to a uniform naming convention.  The application is new and completely rewritten from scratch in TypeScript.  I will be looking at how to integrate this back into the project.

**Comment by Glavin001** [comment]
Awesome! 🎉 

> Pretty Diff version 100

Is it really version 100? I was expecting something like version 3.

**Comment by prettydiff** [comment]
Yeah, so back when NPM3 was unstable and kept breaking my package I closed out that namespace by pushing the version number up to 99.  This was before I created the prettydiff2 namespace.  The only way to use that original namespace is to push the version number above major version 99.

**Comment by prettydiff** [comment]
In semver major versions should be reserved for breaking changes.  This update is certainly a breaking change as I changed the API.  My major version options with NPM are to publish version 100 in the *prettydiff* package name, version 3 in the *prettydiff2* package name, or create a new package name like *prettydiff3*

**Comment by Glavin001** [comment]
> In semver major versions should be reserved for breaking changes.

Yes, of course.

> I closed out that namespace by pushing the version number up to 99. 

Was the version 99 published a real version of Pretty Diff at all? Sounded like it was simply a way close out the namespace -- although I do not understand why it was necessary 🤷‍♂️ . I figured you could unpublish these fake v99 releases and create v3 as the new highest major, but I digress.

> or create a new package name like prettydiff3

👎 

> version 3 in the prettydiff2 package name,

I think publishing `3.0.0` to `prettydiff2` package sounds acceptable then, such that the version number on npm actually reflects the Pretty Diff version number, which I presume is v3?

**Comment by prettydiff** [comment]
I was not aware that versions could be unpublished.  I know that used to be possible, but I thought NPM eliminated that for security reasons.  Yeah, 3.0.2 would be the most accurate version number.
